### ISSUE TITLE
fix measurements() performance for large datasets

### ISSUE THREAD

**Issue by pieterprovoost** [iobis/robis#61]

**Comment by pieterprovoost** [comment]
Switch to `tidyr::unnest_legacy()`:

```
df <- occurrence("Abra nitida", mof = TRUE) %>%
  select(id, mof) %>%
  filter(!sapply(mof, is.null))

microbenchmark::microbenchmark(
  tidyr::unnest(df),
  tidyr::unnest_legacy(df)
)
```

```
Unit: milliseconds
                     expr       min        lq     mean    median        uq       max neval
        tidyr::unnest(df) 3467.9823 3795.0785 4039.506 4065.7586 4184.8828 6116.0731   100
 tidyr::unnest_legacy(df)  112.1484  132.5131  184.605  186.7654  207.3906  847.6721   100
```

**Comment by pieterprovoost** [comment]
Fixed in version 2.2.2 https://github.com/iobis/robis/commit/9b5d5647d6a9489f5457af0b8f45f700fe876782
